### ISSUE TITLE
whatsapp stopped working (related to go-whatsapp)

### ISSUE THREAD

**Issue by BenjaminBeichler** [42wim/matterbridge#1061]
**Describe the bug**
In essence, matterbridge is effected by https://github.com/Rhymen/go-whatsapp/issues/305

which results into 
```
FATAL main:         Starting gateway failed: Bridge whatsapp.bridge failed to start: error decoding
login resp: json: cannot unmarshal arrray into Go value of type map[string]interface {}
```

**To Reproduce**
Configure wa-bridge and try to start

**Expected behavior**
wa should work

**Screenshots/debug logs**
see above

**Environment (please complete the following information):**
 - OS: Linux
 - Matterbridge version: version: 1.17.0 cca1ea24

**Comment by 42wim** [comment]
Should be fixed via workaround in v1.17.1

**Comment by BenjaminBeichler** [comment]
Yes, it works again! Thanks
